Cargo.toml

```diff
@@ -28,14 +28,15 @@ ttf-parser = { version = "0.20.0", default-features = false }
 unicode-linebreak = "0.1.5"
 unicode-script = "0.5.5"
 unicode-segmentation = "1.10.1"
+rayon = { version = "1", optional = true }
 
 [dependencies.unicode-bidi]
 version = "0.3.13"
 default-features = false
 features = ["hardcoded-data"]
 
 [features]
-default = ["std", "swash", "fontconfig"]
+default = ["std", "swash", "fontconfig", "rayon"]
```

Comment:
I would either Place Rayon under the Std feature, OR not add it to defaults and add the std as its own requirement.

Personally, adding it under std seems to be a decent choice here if you want it defaulted.

Reply:
I agree